src/storage.ts

```diff
@@ -158,9 +158,7 @@ class GoogleCloudStorageHandler implements IPackageStorageManager {
       try {
         await file.save(this._convertToString(metadata), {
           validation: this.config.validation || defaultValidation,
-          // Disable resumable uploads to avoid random? failures from google cloud
-          // @see https://stackoverflow.com/questions/53172050/google-cloud-storage-invalid-upload-request-error-bad-request
-          resumable: false
+          resumable: this.config.resumable
```

Comment:
Default is missing (see previous line for an example).

Reply:
I’ve skipped default since google cloud storage will set it to true by default.

This is done to keep previous behavior same by default.

Reply:
Could you add a comment saying the default behavior is `true`. My request is mostly I'm sure someone will ask soon or later.

Reply:
Done